frontend/test/simulate/artifacts/ClusterMutation/some-immutable-field-checks/expected-errors.txt

```diff
@@ -1,12 +1,15 @@
-InvalidRequestContent: properties.platform.networkSecurityGroupId: Field 'networkSecurityGroupId' cannot be updated
-InvalidRequestContent: properties.platform.subnetId: Field 'subnetId' cannot be updated
-InvalidRequestContent: properties.platform.managedResourceGroup: Field 'managedResourceGroup' cannot be updated
-InvalidRequestContent: properties.etcd.dataEncryption.customerManaged.kms.activeKey.name: Field 'name' cannot be updated
-InvalidRequestContent: properties.etcd.dataEncryption.customerManaged.kms.activeKey.vaultName: Field 'vaultName' cannot be updated
-InvalidRequestContent: properties.etcd.dataEncryption.customerManaged.kms.activeKey.version: Field 'version' cannot be updated
```

Comment:
notice 
> InvalidRequestContent: properties.etcd: properties.etcd: Forbidden: field is immutable

in the diff.  We have to decide whether these field are immutable for every type used under dataencryption no matter where the type is used or if the immutability is property of how the type is used here.

Similar for several other fields.  This was conflated before.